hota/mod.json

```diff
@@ -6,10 +6,11 @@
 	"contact" : "http://forum.vcmi.eu/viewtopic.php?t=748",
 	"licenseName" : "Creative Commons Attribution-ShareAlike",
 	"licenseURL" : "http://creativecommons.org/licenses/by-sa/4.0/deed",
-	"version" : "1.6.12",
+	"version" : "1.6.13",
 	"downloadSize" : 114.9,
 	"changelog" : 
 	{
+		"1.6.13"   : [ "Removed unavailable LAND type from terrains." ],
```

Comment:
I think this PR gives zero improvement (and changes) to the players, so why we should show them update available and ask to download 100+Mb?

Reply:
Improvement is that there is no strange warning in client console.
Proposal on how to handle this?

Reply:
But why do we have warning? Could you please paste the warning?

Reply:
Was something like element type 0 not supported blah blah....

![image](https://github.com/vcmi-mods/horn-of-the-abyss/assets/15194321/fb45cb07-23f5-41ea-b489-1409b081d4c4)

Reply:
Okay, I see. We assume that the terrain is land if it's not water.

```
bool TerrainType::isLand() const
{
	return !isWater();
}

```
I can hardly imagine both LAND and WATER flags at same time, but who knows :P

Reply:
What about `clouds`?